routines.R

```diff
@@ -77,3 +77,20 @@ check_if_move_needed <- function(nickname, game) {
   }
   return(output)
 }
+
+put_variables_in_URL <- function(game_uuid, player_uuid, nickname) {
+  updateQueryString(paste0("?game_uuid=", game_uuid, "&player_uuid=", player_uuid, "&nickname=", nickname))
+}
+
+# When putting variables in the URL, NULL variables are converted to empty parameters. We have to convert them back to NULL.
+make_null_from_empty <- function(list) {
+  return(
+    lapply(list, function(x) {
+      if (x == "") {
+        return(NULL)
+      } else {
+        return(x)
```

Comment:
Not sure if I understand -
updateQueryString will encode NULLs as empty strings and strings as non-empty strings.
Therefore, parseQueryString will never give us a NULL - only an empty or a non-empty string. 
In general, I don't think there is a way of making parseQueryString get a NULL (and feed it into the function we're discussing)

Reply:
To be clear - I think `return(x)` will not return a NULL since I think there is no way of specifying the URL in such a way that parseQueryString returns a NULL (and feeds it into this function).

Do you mean that we should check whether `x` is a string vs e.g. a number?

Reply:
In this case, we can just rename the function to something unambiguous like `map_empty_strings_to_null`. If it was indeed a number or anything else, then there's no promise that the function will do anything.